### PR TITLE
Created global exceptions handler

### PR DESCRIPTION
**Pull request by sleepergoose**
1. Created global exceptions handler in the Shared solution (path:  backend\Shared\Shared\ExceptionsHandler), because it is used in both projects. Added some custom exceptions.
2. Included global exceptions handler in Startup.cs both projects:
    using Shared.ExceptionsHandler.Filters;
    ...
    services.AddControllers(options => options.Filters.Add(new CustomExceptionFilterAttribute()));

### CODE REVIEW

File: backend/Perflow/Perflow.csproj [hunk 1]
```diff
@@ -26,10 +26,10 @@
     <Folder Include="Infrastructure\" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Reference Include="Shared">
-      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
-    </Reference>
+  <ItemGroup>
+    <Reference Include="Shared">
+      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Perflow.csproj]: I'm not sure it will work for production, since 'Debug' folder is hard coded

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

File: backend/Perflow/Perflow.csproj (hunk 1 above)
Comment (sleepergoose) [reply backend/Perflow/Perflow.csproj]: Yes, I'm not sure, too. Can you advise me something, please? I thought this reference would be overridden later before production

Comment (sleepergoose) [reply backend/Perflow/Perflow.csproj]: All done. Project reference has been changed.

File: backend/Perflow.Studio/Perflow.Studio.csproj [hunk 2]
```diff
@@ -22,4 +23,10 @@
     <Folder Include="Common\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Shared">
+      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
```
Comment (NikitaKrasnovSW) [inline backend/Perflow.Studio/Perflow.Studio.csproj]: Fix the reference here as well

File: backend/Perflow/Perflow.csproj [hunk 3]
```diff
@@ -25,11 +25,9 @@
...
-  <ItemGroup>
-    <Reference Include="Shared">
-      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
-    </Reference>
+  
+  <ItemGroup>
+    <ProjectReference Include="..\Shared\Shared\Shared.csproj" />
   </ItemGroup>
-
+
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Perflow.csproj]: Delete empty line

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

**Review by NikitaKrasnovSW — APPROVED** [review]

**Pull request merged**